src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -1750,7 +1750,7 @@ public PagedAsyncListCompletionHandler(Class<T> clazz, String name) {
                 }
                 return values;
             }
-            throw new ZendeskResponseException(response);
+            throw ZendeskResponseException.fromResponse(response);
```

Comment:
Instead of throwing the exception for a list, I think it could be better to return an empty list, and treat this case as a normal case.